TombEngine/Objects/TR5/Object/tr5_raisingblock.cpp

```diff
@@ -38,6 +39,22 @@ void InitializeRaisingBlock(short itemNumber)
 	TEN::Collision::Floordata::UpdateBridgeItem(itemNumber);
 }
 
+void ShakeRaisingBlock(ItemInfo* item)
+{
+	SoundEffect(SFX_TR4_RAISING_BLOCK, &item->Pose);
+
+	if (item->TriggerFlags == 0)
+		return;
+
+	if ((item->Pose.Position.ToVector3() - Camera.pos.ToVector3()).Length() < BLOCK(10))
+	{
+		if (item->ItemFlags[1] == 64 || item->ItemFlags[1] == 4096)
```

Comment:
Should these be bitwise flag checks, or are plain comparisons necessary for correct behaviour?

Reply:
I have no idea. It seems it is a remnant of the decompiled code from tomb4 or tomb5.

Reply:
these make no sense to me either